bin/circle/install-circle-predependencies.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Add google-chrome source
+wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
+sudo sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
```

Comment:
I'm fine with that, but note that it's a behavior change: I believe the current command is `>>` appending to google.list, where `tee` with no flags overwrites the file.  Seems fine to me, just making sure it's known.